internal/scan/scanner.go

```diff
@@ -157,8 +161,19 @@ func (dScanner *DebrickedScanner) scanResolve(options DebrickedOptions) error {
 	return nil
 }
 
+type BillingPlanError struct {
+	errorMessage string
+}
+
+func (billingPlanError BillingPlanError) Error() string {
+	return billingPlanError.errorMessage
+}
+
 func (dScanner *DebrickedScanner) scanFingerprint(options DebrickedOptions) error {
 	if options.Fingerprint {
+		if !dScanner.IsEnterpriseCustomer() {
+			return BillingPlanError{errorMessage: "Could not validate Enterprise billing plan, which is a requirement for fingerprinting and manifestless matching."}
```

Comment:
Maybe this could simply state that debricked scan is run without file fingerprinting since no enterprise billing plan exists?